components/service/fretboard/src/test/java/mozilla/components/service/fretboard/storage/flatfile/FlatFileExperimentStorageTest.kt

```diff
@@ -125,4 +125,17 @@ class FlatFileExperimentStorageTest {
         assertEquals(0, experimentsResult.experiments.size)
         assertNull(experimentsResult.lastModified)
     }
+
+    @Test
+    fun testReadingCorruptJSON() {
+        val file = File(RuntimeEnvironment.application.filesDir, "corrupt-experiments.json")
+        file.writer().use {
+            it.write("""{"experiment":[""")
```

Comment:
Now, it's not meaningful but In the future, if we're going to test with the same JSON input in multiple test classes, instead of creating it manually, we could have a single JSON file in assets that can reuse in multiple tests this way when we want to change the JSON we only have one point of change to change.